frontend/src/components/dashboard/FilterChip.vue

```diff
@@ -103,6 +109,16 @@ const filterChipText = computed(() => {
 function mapOption(op: string) {
   return props.alias[op]
 }
+
+function showWarning(show: boolean = true) {
+  warningShowing.value = show
+  // This timing must match the length of the animation in .chip-dropdowm-warning.
+  if (show)
+    warningTimerId = window.setTimeout(() => {
+      warningShowing.value = false
+    }, 3500)
+  else if (warningTimerId) window.clearTimeout(warningTimerId)
```

Comment:
I don't think we need to clear the timeout, since we'll be overriding the existing value of `warningShowing`.

Reply:
We need to clear it because if you quickly select another and then try to deselect, the warning will show again but then quickly vanish since the timeout was still running and will trigger.